docs/README.md

```diff
@@ -1,34 +1,31 @@
-# KubeVela Documentation
-
-Learn and use KubeVela with tutorials and user stories.
-
-**Overview**
-- [Introduction](introduction.md)
-- [Installation](./install.md)
-- [Quick Start](quick-start.md)
-- [Concepts](concepts.md)
-
-**For Developers**
-- [Setting Up Deployment Environment](developers/config-enviroments.md)
-- [Initializing Application](developers/app-init.md)
-- [Setting Routes](developers/set-route.md)
-- [Setting Auto-scaling Policy](developers/set-autoscale.md)
-- [Setting Rollout Strategy](developers/set-rollout.md)
-- [Monitoring Application](developers/set-metrics.md)
-- [Using Appfile](developers/devex/appfile.md)
-- [Check Application Logs](developers/check-logs.md)
-- [Execute Commands in Container](developers/exec-cmd.md)
-- [Port Forward to Container](developers/port-forward.md)
-- [Configuring data/env in Application](developers/config-app.md)
-- [Consuming Cloud Services](developers/cloud-service.md)
-- [Managing Capabilities](developers/cap-center.md)
-- [Capability References](developers/references/README.md)
-
-**For Platform Engineers**
-- [Extending KubeVela](platform-engineers/extending-kubevela.md)
-
-**Internals**
-- [Design and Architecture](design.md)
-
-**Roadmap**
-- [Roadmap of KubeVela](roadmap.md)
\ No newline at end of file
+# KubeVela Website
+[kubevela.io](https://kubevela.io)
+
+## Contributing
+
+- Fork and clone
+
+  ```
+  $ git clone git@github.com:oam-dev/kubevela.git
```

Comment:
Nit. If the repo is forked, the command should be `git clone git@github.com:github-id/kubevela.git`.

Reply:
Per the discussion with @wonderflow , it's okay to keep your version. Thanks.